### DISCUSSION TITLE
Is it free for personal use?

### DISCUSSION THREAD

**Discussion by bbhxwl** [community/community#16697] [Codespaces]
Is there a limit on how many warehouses and how much running time?

**Comment by alexavil** [comment]
During the beta it appears to be free, but I'm curious about billing after the beta. Will it be restricted to a subscription, and/or will we have to pay hourly, like orgs?

**Reply by bbhxwl** [reply]
> During the beta it appears to be free, but I'm curious about billing after the beta. Will it be restricted to a subscription, or will we have to pay hourly, like orgs?

I also use gitpod now. I can accept this charging method, unlimited hours, but there is no server in Asia.

**Comment by darthwalsh** [comment]
The [existing "viewing usage" docs](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/viewing-your-codespaces-usage) haven't been updated for Codespaces usage in personal repos -- maybe if I converted `github.com/darthwalsh` into an Organization I could see Codespaces use on my "org's" repos, but I won't actually make that change.

I set my codespace timeout to 5 minutes and I'll assume it's free for now, and maybe I'll get a bill for a few dollars some month and I'll reconsider.

**Reply by alexavil** [reply]
In fact, comparing plans right now shows that Codespaces aren't available in any plan.
I wonder if that's a leftover, or it will take effect after the beta?
![изображение](https://user-images.githubusercontent.com/32246131/168881748-62044f55-77bc-4deb-a531-74203edd4dd7.png)

**Comment by dmilburn** [comment]
Hi from the codespaces team! yes, codespaces is currently free for individual use if you're part of the beta. Can you explain more what you mean by how many warehouses? We'll give you in app notifications for any possible limits like having too many concurrent codespaces.
